plugins/modules/aci_fabric_node.py

```diff
@@ -243,10 +258,19 @@ def main():
     switch = module.params.get("switch")
     description = module.params.get("description")
     role = module.params.get("role")
+    node_type = module.params.get("node_type")
+    remote_leaf_pool_id = module.params.get("remote_leaf_pool_id")
     state = module.params.get("state")
     name_alias = module.params.get("name_alias")
 
     aci = ACIModule(module)
+
+    if node_type is not None:
+        node_type = NODE_TYPE_MAPPING.get(node_type)
```

Comment:
NODE_TYPE_MAPPING = {"tier-2": "tier-2-leaf", "remote": "remote-leaf-wan", "virtual": "virtual", "unspecified": "unspecified"} does not have a key called node_type?

Reply:
Ignore the comment. I didn't see the choices